optimum/commands/export/onnx.py

```diff
@@ -150,6 +150,12 @@ def parse_args_onnx(parser):
             "Also disable the use of position_ids for text-generation models that require it for batched generation. This argument is introduced for backward compatibility and will be removed in a future release of Optimum."
         ),
     )
+    optional_group.add_argument(
+        "--fuse_lora",
+        type=str,
+        default=None,
+        help=("For Stable Diffusion models only. LoRA to fuse before export."),
```

Comment:
Could you precise what kind of lora checkpoints are supported, what needs to be passed here? A link to diffusers/whatever documentation would be nice if there exists one.

Reply:
I'd like to point that i have been able to use this patch to fuse LCM into SD and then exporting it to ONNX. tested it using `pipe LCMScheduler` loading the individual ONNX models and works. in that way, now i can rewrite LCMScheduler for ONNX and have 4 steps ineference on ONNX